make.ps1

```diff
@@ -47,6 +47,12 @@ if ($command -eq "all")
 	}
 	else
 	{
+		cd thirdparty
```

Comment:
If we go ahead with this idea then this should call `make dependencies` instead of copying the commands.